### PR TITLE
[ADD] legal: terms of sales

### PR DESCRIPTION
**Pull request by dbo-odoo**
Task: https://www.odoo.com/web?debug=#id=31917&view_type=form&model=project.task&action=327&menu_id=4720

- rst web page
- latexpdf files & config

### CODE REVIEW

**Comment by dbo-odoo** [comment]
I don't think we want to crowd-source our legal documents
:question:

**Comment by mart-e** [comment]
We can make translations and no publish it on transifex

**Comment by dbo-odoo** [comment]
As you wish then, maybe to check with @odony - I copied the way the Enterprise Agreement works at the moment

**Comment by mart-e** [comment]
[legal.pot](https://gist.github.com/mart-e/ad80ec97754384923e7a917b21a4ec28)

**Comment by dbo-odoo** [comment]
legalize it!

**Comment by dbo-odoo** [comment]
@mart-e : will the legal.pot file with latex2pdf as well?

**Comment by mart-e** [comment]
We may need to do something like c0098583eb4c to make it work.

**Comment by dbo-odoo** [comment]
@mart-e 
So if we don't do the l10n, seems good for you?

**Comment by mart-e** [comment]
If I wanted to be touchy, I would ask to keep 80 char per line but sure, looks good

**Comment by dbo-odoo** [comment]
@mart-e 
Done

**Comment by dbo-odoo** [comment]
@mart-e 
Are latxetpdf builds made daily as well?

File: legal/terms/terms_of_sale.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+In order for it to be admissible, Odoo S.A. must be notified of any claim by
+means of a letter sent by recorded delivery to its registered office within 8
+days of the delivery of the goods or the provision of the services.
+
+To the maximum extent permitted by law, the aggregate liability of each party
+together with its affiliates arising will not exceed 50% of the total amount
+paid by the Customer under this Agreement during the 12 months immediately
+preceding the date of the event giving rise to such claim. Multiple claims shall
+not enlarge this limitation.
```
Comment (odony) [inline legal/terms/terms_of_sale.rst]: - provision for updates
- version info
- section about conflict with specific agreement
- "agreement" -> conditions
- perhaps short intro: 

> These Conditions of Sale govern the sale of products and services by Amazon EU SARL to you. We offer a wide range of Amazon Services, and sometimes additional terms may apply. When you use an Amazon Service (for example Your Profile, Gift Cards, Amazon applications for mobile or Message Centre), you will also be subject to the terms, guidelines and conditions applicable to that Amazon Service ("Service Terms"). If these Conditions of Sale are inconsistent with the Service Terms, those Service Terms will control.

Please read these conditions carefully before placing an order with Amazon EU SARL. By placing an order with Amazon EU SARL, you signify your agreement to be bound by these conditions.

Comment (odony) [inline legal/terms/terms_of_sale.rst]: Links to FR/EN terms of amazon:
- https://www.amazon.co.uk/gp/help/customer/display.html?nodeId=201909000
- https://www.amazon.fr/gp/help/customer/display.html?nodeId=548524

**Comment by dbo-odoo** [comment]
@odony 

modifs done, tu sais jeter un oeil sur mon legal-speak?

File: legal/terms/i18n/terms_of_sale_fr.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+Ces Conditions Générales de Vente régissent la vente de produits et service par
+Odoo S.A. à vous. Des conditions supplémentaires peuvent s'appliquer pour
+d'autres services offerts par Odoo S.A. (par exemple, l'Odoo Enterprise
+Subscription Agreement). Si ces conditions supplémentaires sont incompatibles
+avec les Conditions Générales de Vente, ces conditions supplémentaires
+supplantent les Conditions Générales.
+
+Veuillez lire ces conditions attentivement avant de confirmer une offre avec
+Odoo S.A.. En confirmant une offre avec Odoo S.A., vous marquez votre accord
+avec ces conditions.
```
Comment (odony) [inline legal/terms/i18n/terms_of_sale_fr.rst]: Proposition de variation subtile pour les 2 premiers paragraphes:

> Ces conditions régissent la vente de produits et services entre
> Odoo S.A. et vous. Des conditions supplémentaires peuvent s'appliquer pour
> d'autres services offerts par Odoo S.A. (par exemple, l'Odoo Enterprise
> Subscription Agreement). Si ces conditions supplémentaires entrent en contradiction
> avec les Conditions Générales de Vente, les conditions supplémentaires prévaudront.

> Veuillez lire ces conditions attentivement avant de faire une commande
> auprès d'Odoo S.A.. En commandant auprès d'Odoo S.A., vous marquez votre
> accord avec les présentes conditions.

File: legal/terms/i18n/terms_of_sale_fr.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+Odoo S.A. s'engage à faire ses meilleurs efforts en vue de fournir en temps
+utile des services performants. Toutefois, aucune de ses obligations ne peut
+être considérée comme étant une obligation de résultat. Odoo S.A. ne peut, en
+aucun cas, être appelée en garantie par le client dans le cadre de demandes en
+dommages et intérêts formulée par le consommateur final à l'encontre du client.
+
+Dans la limite autorisée par la loi, la responsabilité globale de chaque partie,
+ainsi que de ses filiales ne dépassera pas 50% du montant total réglé par le
+Client en vertu des présente Conditions au cours des 12 mois précédant la date
```
Comment (odony) [inline legal/terms/i18n/terms_of_sale_fr.rst]: je ne capitaliserais pas "présente Conditions" -> "présente**s** conditions", sinon il faut mettre CGV en entier. Je ne pense pas que ça prête à confusion ici, donc pas nécessaire

File: legal/terms/terms_of_sale.rst [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+General Terms of Sale
+=====================
+
+.. note:: Last revision: May 3, 2017.
+
+These General Terms of Sale govern the sale of products and services by
+Odoo S.A. to you. Additional terms may apply for services provided by Odoo S.A.
+(for example, the Enterprise  Subscription Agreements). If these additional
+terms are inconsistent with the General Terms of Sale, the additional terms will
+supersede these terms.
```
Comment (odony) [inline legal/terms/terms_of_sale.rst]: I would opt for "the additional terms will prevail over these General Terms of Sale."
Subtle distinction, I know ;-)
Interestingly, [this author](http://www.adamsdrafting.com/control-prevail-take-precedence-or-govern/) also picked "prevail" after comparing some Amazon and Google TOS  ;-)

File: legal/terms/terms_of_sale.rst [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+=====================
+General Terms of Sale
+=====================
+
+.. note:: Last revision: May 3, 2017.
+
+These General Terms of Sale govern the sale of products and services by
+Odoo S.A. to you. Additional terms may apply for services provided by Odoo S.A.
+(for example, the Enterprise  Subscription Agreements). If these additional
```
Comment (odony) [inline legal/terms/terms_of_sale.rst]: extra space

File: legal/terms/terms_of_sale.rst [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+terms are inconsistent with the General Terms of Sale, the additional terms will
+supersede these terms.
+
+Please read these terms carefully before placing an order with Odoo S.A.. By
```
Comment (odony) [inline legal/terms/terms_of_sale.rst]: double period signs? I think we can omit the ending period in this case.

**Comment by dbo-odoo** [comment]
Merged in 9.0, cherry-picked in 10.0 and 11.0.

**Pull request merged**